### ISSUE TITLE
A11y feedback

### ISSUE THREAD

**Issue by NoelDeMartin** [SolidOS/profile-pane#310]
I've taken a quick look at the profile-pane accesibility, and here's some things that could be improved:

- **Remove sr-only h1/h2 headings.** There are two headings at the beginning that I don't think are necessary. In fact, the h1 should be removed for sure because there should only be a single `<h1>` element per page.
- **Improve `<aside>` headings and location.** I'm not sure the `<aside>` element is [placed correctly](https://stackoverflow.com/questions/38023089/should-i-have-aside-element-outside-or-inside-of-main-element). Additionally, the heading labeling the section should be a level higher than the headings inside. Instead, they are all `<h2>`. The children should be rendered as `<h3>` instead.
- **There are two `<main>` elements.** There should only be one per page.
- **`<table>` elements aren't using the correct role.** When tables are used for layout purposes, they should use the `role="presentation"` attribute. Otherwise, they can be confused by assistive technologies as data tables and interfere with the navigation.
- **Hidden content.** There is some content hidden from screen readers that is visible for sighted users, which may create some confusion. For example, in case the skills section is empty, the text "No skills added yet" is hidden from screen readers. Instead, the entire section should be hidden, or we shouldn't hide anything.

**Comment by m5x5** [comment]
@NoelDeMartin are you planning to work on the E2E tests in the near future? Would you rather pick Cypress or Playwright?
I now also need E2E tests because if I make a change with the main elements, I will have to verify all pages have exactly one main element.
I could already write a local E2E test in the format of the framework that you're planning to use.

**Comment by NoelDeMartin** [comment]
@m5x5 Hey thanks, I guess we have to talk about it with the rest of the team, but yes it's very likely that I also end up configuring some tests on my own xD. I agree with using Playwright, I've also started migrating to that from Cypress in my other projects :).

For this particular issue, though, I'm not sure how easy it will be to make it work. Because as I mentioned on the call, I'm not sure the first `<main>` even comes from this package :(. But if you find an easy way to get it done, and don't have to dedicate too much time to that, go ahead and include the tests. We can discuss more about the approach and whether we like it or not when we review the PR looking at the actual code.

Btw, I wrote the feedback looking at this page, not running this component locally or anything. Keep that in mind when you work on this: https://timea3.pivot-test.solidproject.org:3000/profile/card#me

**Comment by m5x5** [comment]
@NoelDeMartin I created three pull requests to fix the issue with the main tags. I decided the app-view would be the better solution than building it together with js for every pane. It now is a sibling of aside, similar to how it was mentioned in the pattern, and also what seemed intuitive to me.
The duplication is also gone now.

**Comment by timea-solid** [comment]
@NoelDeMartin  @m5x5  awesome work! All done.
